app/controllers/agent_controller.rb

```diff
@@ -2,5 +2,8 @@ class AgentController < ApplicationController
   before_action :authenticate_user!
 
   def dashboard
+    @policy_count = Policy.count
```

Comment:
move this query in model because querying from model is faster as compared to controller